src/errors/test-run/templates.js

```diff
@@ -108,6 +139,30 @@ export default {
         ${err.getCallsiteMarkup()}
     `),
 
+    [TYPE.selectActionElementNonEditableError]: err => markup(err, `
+        The action element is expected to be editable (an input, textarea or element with the contentEditable attribute).
+
+        ${err.getCallsiteMarkup()}
+    `),
+
+    [TYPE.selectActionElementNonContentEditableError]: err => markup(err, `
+        The element that matches the specified ${err.argumentTitle} selector is expected to have the contentEditable attribute enabled or the entire document should be in design mode.
```

Comment:
Can't this error be combined with error above? Both of them require `contentEditable`